src/ci/scripts/verify-backported-commits.sh

```diff
@@ -0,0 +1,32 @@
+#!/bin/bash
+# Ensure commits in beta are in master & commits in stable are in beta + master.
+set -euo pipefail
+IFS=$'\n\t'
+
+source "$(cd "$(dirname "$0")" && pwd)/../shared.sh"
+
+cherries_dir="$(dirname "$0")/cherries"
+
+verify_cherry_file() {
+  base_branch=$1
+  current_branch=$2
+  cherry_file="$cherries_dir/$current_branch-$base_branch"
+
+  git cherry "origin/$base_branch" | sort -s -t' ' -k2 > $cherry_file
+
+  if ! git diff --quiet $cherry_file; then
+    echo "Found differing commits between $base_branch and $current_branch."
+    git diff $cherry_file
+    exit 1
+  fi
+}
+
+ci_base_branch=$(ciBaseBranch)
+if [[ $ci_base_branch == "beta" ]]; then
+  verify_cherry_file "master" "beta"
+elif [[ $ci_base_branch == "stable" ]]; then
+  verify_cherry_file "master" "stable"
+  verify_cherry_file "beta" "stable"
```

Comment:
All these quotes are unnecessary (if you're going to quote anything, it should be the variable).
```suggestion
if [[ $ci_base_branch == beta ]]; then
  verify_cherry_file master beta
elif [[ $ci_base_branch == stable ]]; then
  verify_cherry_file master stable
  verify_cherry_file beta stable
```

Reply:
I've included your suggestions - thanks!